Add profile and first blog post for cachebench

haochengxia opened the pull request:
Added the profile and the first blog post for [CacheBench](https://ucsc-ospo.github.io/project/osre25/harvard/cachebench/).

slieggi commented:
@haochengxia - 
1. Your project is NOT GSoC but SoR; please fix tags and all reference to GSoC in the text. You are funded through the OSRE and the SoR. 
2.  You have the incorrect directory path for the blog. It should be under content/report/osre25/harvard/cachebench/2025-06-21-haochengxia/index.md

haochengxia commented:
> @haochengxia -
> 
> 1. Your project is NOT GSoC but SoR; please fix tags and all reference to GSoC in the text. You are funded through the OSRE and the SoR.
> 2. You have the incorrect directory path for the blog. It should be under content/report/osre25/harvard/cachebench/2025-06-21-haochengxia/index.md

Thanks for you kind and detailed instructions. Changed the content on GSoC and moved the file to correct path.
